charts/oidc-webhook-authenticator/values.yaml

```
@@ -9,7 +9,7 @@ replicaCount: 1
 
 image:
   repository: eu.gcr.io/gardener-project/gardener/oidc-webhook-authenticator
-  tag: v0.1.0
+  tag: v0.3.0
```

To avoid updating this on each release, you can use `latest`. It is not recommended, but can be easily overwritten during installation time.
```suggestion
  tag: latest
```

---

Done. I guess only commits merged in the main branch are tagged as latest so it should be fine.

---

It is up to the pipeline configuration which images to be also tagged as latest.